src/index.js

```diff
@@ -172,17 +178,34 @@ const oneAppDevCdnFactory = ({
         remoteModuleBaseUrls
       );
       const remoteModuleBaseUrlOrigin = new URL(knownRemoteModuleBaseUrl).origin;
+      if (cachedModules[incomingRequestPath]) {
+        console.log(`Returning from cached module for path ${incomingRequestPath}`);
```

Comment:
Lets remove this log, its just a lot of clutter and doesn't provide the user any meaningful information